crates/hir-expand/src/db.rs

```diff
@@ -699,9 +700,9 @@ fn expand_proc_macro(
             loc.krate,
             &macro_arg,
             attr_arg,
-            span_with_def_site_ctxt(db, span, id),
-            span_with_call_site_ctxt(db, span, id),
-            span_with_mixed_site_ctxt(db, span, id),
+            span_with_def_site_ctxt(db, span, id, loc.def.edition),
+            span_with_call_site_ctxt(db, span, id, loc.def.edition),
+            span_with_mixed_site_ctxt(db, span, id, loc.def.edition),
```

Comment:
I am not entirely sure this right, need to double check with rustc here

Reply:
Hmm, so rustc only has the concept of the "local crate" here, that is the crate that is being analyzed right now (we can't model it like that). In my eyes, that would imply we ought to fill the call site edition here instead?